src/NUnitFramework/tests/Assertions/AdhocTestExecutionTests.cs

```diff
@@ -3,7 +3,7 @@
 using System.Reflection;
 using NUnit.Framework.Internal;
 
-#if !NETCOREAPP1_1
+#if NET20 || NET35 || NET40 || NET45
```

Comment:
Why not `PLATFORM` as in `AssertThatTests.cs`

Reply:
I'm not seeing anything specific to the NUnit platform-detection feature in this file. In `AssertThatTests` we're using a PlatformAttribute which must be #if'd out in order to compile, so O used `#if PLATFORM` there because the definition of PlatformAttribute itself exists iff `PLATFORM` is defined.

Reply:
Ok. I think I understand it now (and misunderstod the `PLATFORM` define). I'll delete most of my comments below 😄 .